scousepy/stage_1.py

```diff
@@ -66,9 +66,11 @@ def compute_noise(scouseobject):
         stopcount+=1
         specidx+=1
 
-    rms = np.median(rmsList)
-
-    return rms
+    if np.all(~np.isfinite(np.asarray(rmsList))):
+        raise ValueError("All RMS are NaN, which is not allowed.")
```

Comment:
it would be good to highlight some known scenarios that can produce this, or point to a docs page where we enumerate those. We've effectively gotten to this case once already, right?